### DISCUSSION TITLE
Where can I find the latest docs?

### DISCUSSION THREAD

**Discussion by andrewjmac** [Comcast/react-data-grid#2711] [Q&A]
Can't find the docs to save my life, i've found this: https://adazzle.github.io/react-data-grid/#/common-features 
But where is the actual code behind??

Many thanks

**Comment by mteason** [comment]
You can clone the project and the demo site is in the website folder. There are code what you need.

**Comment by andrewjmac** [comment]
Thank you for the reply, do you know what version that’s running off? I’m trying to use the beta7 one and I’ve been looking through the source on GitHub and trying to re-crest, but unfortunately as of yet it’s not working 🙃😦

On 30 Oct 2021, at 04:19, mteason ***@***.***> wrote:

﻿

You can clone the project and the demo site is in the website folder. There are code what you need.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/adazzle/react-data-grid/discussions/2711#discussioncomment-1561835>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ACE6XWLP4NW5A5NURBD3WNTUJNP3LANCNFSM5GQ263QQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by mteason** [reply]
Check the version at package.json.
